src/bin/pg_autoctl/primary_standby.c

```diff
@@ -411,7 +411,17 @@ primary_add_standby_to_hba(LocalPostgresServer *postgres, char *standbyHostname,
 	PGSQL *pgsql = &(postgres->sqlClient);
 	PostgresSetup *postgresSetup = &(postgres->postgresSetup);
 	char hbaFilePath[MAXPGPATH];
-	char* authMethod = replicationPassword ? "md5" : "trust";
+	char *authMethod = NULL;
```

Comment:
I think it would be easier to read that way:

``` C
	char *authMethod = "trust";

	if (replicationPassword)
	{
		authMethod = pg_setup_get_auth_method(postgresSetup);
	}    
```

Reply:
ok